src/Microsoft.AspNetCore.Razor.Language/Legacy/AddTagHelperChunkGenerator.cs

```diff
@@ -63,5 +64,28 @@ public override int GetHashCode()
 
             return combiner.CombinedHash;
         }
+
+        public override string ToString()
+        {
+            var builder = new StringBuilder("AddTagHelper:{");
```

Comment:
Decided to do these here rather than in `SyntaxTreeNodeWriter` so we get this during debugging as well.